src/ansi_test.go

```diff
@@ -26,7 +26,7 @@ func TestExtractColor(t *testing.T) {
 		output, ansiOffsets, newState := extractColor(src, state, nil)
 		state = newState
 		if output != "hello world" {
-			t.Errorf("Invalid output: %s %s", output, []rune(output))
+			t.Errorf("Invalid output: %s %v", output, []rune(output))
```

Comment:
Apparently, you can't use `%s` with a rune slice.
`%v` will print `[]rune("hello")` like `[]rune{'h', 'e', 'l', 'l', 'o'}`.